WBooks/DetailBook/DetailBookViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  DetailBookViewController.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 25/08/21.
+//
+
+import UIKit
+
+class DetailBookViewController: UIViewController {
+    
+    private lazy var detailBookView = DetailBookView()
+    private let detailBookViewModel: DetailBookViewModel
+
+    init(detailBookViewModel: DetailBookViewModel) {
+        self.detailBookViewModel = detailBookViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = NSLocalizedString("TITLE_DETAILBOOK", comment: "")
+        setupTable()
+        setupView()
+        setupActions()
+        getComments()
+    }
+    
+    override func loadView() {
+        view = detailBookView
+    }
+    
+    func setupTable() {
+        detailBookView.tableView.delegate = self
+        detailBookView.tableView.dataSource = self
+    }
+    
+    func setupView() {
+        let book = detailBookViewModel.book
+        detailBookView.bookImage.imageFromUrl(urlString: book.image)
+        detailBookView.titleLabel.text = book.title
+        detailBookView.authorLabel.text = book.author
+        detailBookView.yearLabel.text = book.year
+        detailBookView.genreLabel.text = book.genre
+        detailBookView.statusLabel.text = book.status
+    }
+    
+    func setupActions() {
+        detailBookView.rentButton.addTarget(self, action: #selector(rentPress), for: .touchUpInside)
+        detailBookView.addWishlistButton.addTarget(self, action: #selector(addWishlistPress), for: .touchUpInside)
+    }
+    
+    func getComments() {
+        detailBookViewModel.getComments { comments in
+            if comments.count > 0 {
```

Comment:
A little improvement is to use isEmpty instead of count > 0 
https://betterprogramming.pub/strings-comparison-isempty-vs-count-0-be80d701901b

Reply:
The comparison in this case is made for an array